Modules/Loadable/VolumeRendering/MRML/vtkMRMLGPURayCastVolumeRenderingDisplayNode.h

```diff
@@ -81,14 +77,8 @@ class VTK_SLICER_VOLUMERENDERING_MODULE_MRML_EXPORT vtkMRMLGPURayCastVolumeRende
    * */
   int RaycastTechnique;
 
-  /// Use jittering to reduce the wood-grain effect if on. Off by default
-  int UseJittering;
-
-  /// Compute sample distance from volume spacing if on. Off by default
-  /// If turned on, then AutoAdjustSampleDistances and explicit SampleDistance
-  /// will be ignored, and only the internal auto-calculation will determine
-  /// the sampling distance.
-  int LockSampleDistanceToInputSpacing;
+  /// Make surface appearance smoother. Off by default
```

Comment:
I think the part of the comment about the wood-grain atrifact is meaningful. I don't think making the documentation more vague is desirable.

Reply:
I moved these details to the get/set method description so that is included in the API documentation.

We can include details here as well, but I thought it is easier to maintain documentation if there is less duplication.

Reply:
OK no problem. I didn't see the set/get part, just saw that this info was not there anymore.